src/Command/Project/ProjectCreateCommand.php

```diff
@@ -216,29 +258,57 @@ protected function getAvailableRegions($runtime = false)
         return $regions;
     }
 
+    /**
+     * Return the catalog.
+     *
+     * The default list is in the config `service.catalog`. This is
+     * replaced at runtime by an API call.
+     *
+     * @param bool $runtime
+     *
+     * @return array
+     */
+    protected function getAvailableCatalog($runtime = false)
+    {
+        if ($runtime) {
+            $catalog = [];
+            foreach ($this->api()->getClient()->getCatalog()->getData() as $item) {
```

Comment:
This needs to handle `getCatalog()` returning no items, or returning no items that contain an 'info' and a 'template'.